content/docs/envoy/main/integrations/aws-elb/alb.md

```diff
@@ -12,20 +12,42 @@ The AWS Load Balancer Controller only supports the creation of an ALB through an
 ## Before you begin
 
 1. Create or use an existing AWS account. 
-2. Follow the [Get started guide](/docs/quickstart/) to install kgateway. You do not need to set up a Gateway as you create a custom Gateway as part of this guide.
-3. Follow the [Sample app guide]({{< link-hextra path="/install/sample-app#deploy-app/" >}}) to deploy the httpbin sample app.
+2. Follow the [Get started guide]({{< link-hextra path="/quickstart/" >}}) to install {{< reuse "docs/snippets/kgateway.md" >}}.
+3. Deploy the httpbin sample app. For more information, see the [sample app guide]({{< link-hextra path="/install/sample-app#deploy-app/" >}}).
```

Comment:
they need the sample app but nothing else from the sample app guide, so just put the 2 commands needed here